components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.devices/devices.js

```diff
@@ -23,6 +23,7 @@ function onRequest(context) {
 
     var groupName = request.getParameter("groupName");
     var groupOwner = request.getParameter("groupOwner");
```

Comment:
IMO, we don't need to have groupName and groupOwner in here as they can also remove from the group module. WDYT?